app_legacy/Helpers/database/user.php

```diff
@@ -379,6 +379,17 @@ function GetDeveloperStatsFull(int $count, int $sortBy, int $devFilter = 7): arr
         Ticket AS tick ON (tick.AchievementID = ach.ID AND tick.ReportState IN (" . TicketState::Open . "," . TicketState::Request . "))
     LEFT JOIN
         SetClaim AS sc ON (sc.User = ua.User AND sc.Status = " . ClaimStatus::Active . ")
+    LEFT JOIN (
+        SELECT ua2.User,
+        SUM(CASE WHEN t.ReportState = 2 THEN 1 ELSE 0 END) AS total
+        FROM Ticket AS t
+        LEFT JOIN UserAccounts as ua ON ua.ID = t.ReportedByUserID
+        LEFT JOIN UserAccounts as ua2 ON ua2.ID = t.ResolvedByUserID
+        LEFT JOIN Achievements as a ON a.ID = t.AchievementID
+        WHERE ua.User NOT LIKE ua2.User
+        AND a.Author NOT LIKE ua2.User
+        AND a.Flags = '3' 
+        GROUP BY ua2.User) resolved ON resolved.User = ua.User
```

Comment:
Not sure how much it matters, but this query doesn't result in the same count as the query performed on the linked page.

For example, in my DB, MGNS8M shows 43 in the developer stats page, but 79 when clicked.

Reply:
I listed that as a caveat since the existing individual stats metric excludes tickets you report and the ticket manager didn't have a filter for that but I went ahead and added the filter now since it probably should be added if I'm going to link to to the ticket manager for that